modules/kind/00_mod.mk

```diff
@@ -15,7 +15,14 @@
 images_amd64 ?=
 images_arm64 ?=
 
-kind_version := v1.27.3
+kind_version := v1.29.0
```

Comment:
suggestion (non-blocking): I think this variable is confusingly named because it makes me think we're talking about the version of the `kind` tool.

This is the version of k8s we'll be using with `kind`, right?

Might be worth changing to `kind_k8s_version` or something, but that'd be a separate PR!

Reply:
Yes, the name is confusing, I'll create a PR.